packages/crew_url_opener.rb

```diff
@@ -0,0 +1,56 @@
+require 'package'
+
+class Crew_url_opener < Package
+  description 'Script for opening an URL in Chrome OS browser'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '1.0'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'SKIP'
+  source_sha256 'SKIP'
```

Comment:
I believe you can remove this line and it should still skip the source.

Reply:
Done